modules/API/API.js

```diff
@@ -1451,6 +1453,40 @@ class API {
         });
     }
 
+    /**
+     * Notify external application (if API is enabled) that the prejoin video
+     * visibility had changed.
+     *
+     * @param {boolean} isVisible - Whether the prejoin video is visible.
+     * @returns {void}
+     */
+    notifyPrejoinVideoVisibilityChanged(isVisible: boolean) {
+        this._sendEvent({
+            name: 'on-prejoin-video-changed',
+            isVisible
+        });
+    }
+
+    /**
+     * Notify external application (if API is enabled) that the prejoin
+     * screen was loaded.
+     *
+     * @returns {void}
+     */
+    notifyPrejoinLoaded() {
+        const state = APP.store.getState();
+        const localParticipant = getLocalParticipant(state);
+        const { defaultLocalDisplayName } = state['features/base/config'];
+        const displayName = getDisplayName(state);
+
+        this._sendEvent({
+            name: 'prejoin-screen-loaded',
+            id: localParticipant.id, // the id is 'local' when in prejoin.
```

Comment:
maybe skip this since it can only be yourself?

Reply:
you mean, skip getting the participant and send the LOCAL_PARTICIPANT_ID  instead?

Reply:
I mean to not send any ID whatsoever.